ordered.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Ordered List</title>
+</head>
+<body>
+<h1>Ordered List</h1>
+<h2>ETAPELE ACTIVITĂŢII INFRACŢIONALE</h2>
+<di>
+    <p>
+        Ca act de conduită exterioară, infracţiunea presupune o desfăşurare în spaţiu şi în timp.Acea acţiune sau inacţiune pe care persoana o are la momentul săvîrşirii infracţiunii, fie săvîrşită spontan sau cu premeditare.
+</p>
+</di>
+<h3>Doctrina penală distinge două perioade în care se desfăşoară activitatea infracţională:</h3>
+<ol>
+    <li>Perioada internă a activităţii infracţionale (perioada psihică)-în cadrul acestei perioade se disting 3 momente :
+    <ol>
+        <li> al naşterii sau conceperii ideii de săvîrşire a infracţiunii</li>
+        <li>al deliberării în care persoana cîntăreşte motivele pro şi contra ideii de a săvîrşi o infracţiune.</li>
+        <li>al deciziei sau hotărîrii de a săvîrşi o infracţiune.</li>
+    </ol>
+    </li>
+    <li>
+        Perioada externă <br>
+        <span>Pregătirea de infracţiune sau actele preparatorii este prima fază a perioadei externe a activităţii infracţionale şi constau în anumite acte, activităţi de procurare de date, informaţii ori de adaptare a mijloacelor sau instrumentelor ce vor fi folosite la săvîrşirea infracţiunii sau în activităţi de creare a condiţiilor favorabile săvîrşirii infracţiunii. Aceste acte preparatorii sunt posibile numai la infracţiunile intenţionate.</span>
+      <br> <i><span>Pentru a fi considerate acte de pregătire a infracţiunii activităţile desfăşurate trebuie să îndeplinească anumite condiţii:</span></i>
```

Comment:
aici span poate fi omis si poti pune textul direct in i tag daca vreai sa fie italic

Reply:
done!